DigitalLearningSolutions.Web/Views/TrackingSystem/Delegates/GroupDelegates/ConfirmDelegateAdded.cshtml

```diff
@@ -0,0 +1,25 @@
+﻿@using DigitalLearningSolutions.Web.ViewComponents
+@using DigitalLearningSolutions.Web.ViewModels.TrackingSystem.Delegates.GroupDelegates
+@model ConfirmDelegateAddedViewModel
+
+@{
+  ViewData["Title"] = "Delegate added to group"; 
+
+  var routeParamsForIndexLink = new Dictionary<string, string?> {
+    { "groupId", Model.GroupId.ToString() },
+  };
+}
+
+<div class="nhsuk-grid-row">
+  <div class="nhsuk-grid-column-full">
+    <h1 class="nhsuk-heading-l nhsuk-u-margin-bottom-9">
+      Delegate added to group
+    </h1>
+
+    <p class="nhsuk-body-l nhsuk-u-margin-bottom-8">
+      @(Model.DelegateName) has been added to the @(Model.GroupName) group
+    </p>
+
+    <vc:action-link link-text="See all delegates in @(Model.GroupName) group" asp-controller="GroupDelegates" asp-action="Index" asp-all-route-data="@routeParamsForIndexLink" />

```

Comment:
Might be worth breaking down this long line

Reply:
Done.